From 0232061a1d1cfa28dfc7cdf4615e3c94f868abd0 Mon Sep 17 00:00:00 2001
From: Beniamino Galvani <bgalvani@redhat.com>
Date: Tue, 11 Oct 2016 16:44:21 +0200
Subject: [PATCH] core: downgrade message level for unknown device driver

It's normal that the driver can't be determined for some device types
that we don't manage, for example tun devices. Downgrade the log level
to DEBUG for such messages.

 /sys/devices/virtual/net/virbr0-nic: couldn't determine device driver; ignoring...

https://bugzilla.redhat.com/show_bug.cgi?id=1331314
---
 src/nm-udev-manager.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/nm-udev-manager.c b/src/nm-udev-manager.c
index dfe862a..0f80f5e 100644
--- a/src/nm-udev-manager.c
+++ b/src/nm-udev-manager.c
@@ -438,7 +438,7 @@ net_add (NMUdevManager *self, GUdevDevice *device)
 		}
 		
 		if (!driver) {
-			nm_log_warn (LOGD_HW, "%s: couldn't determine device driver; ignoring...", path);
+			nm_log_dbg (LOGD_HW, "%s: couldn't determine device driver; ignoring...", path);
 			goto out;
 		}
 	}
-- 
2.7.4

